scripts/get-test-pass-rate.mjs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+// Licensed under the MIT License.
+
+// This script is used in the "runAfterAll" stage in our E2E test pipeline. It's used
+// to get timeline and metrics data so that the test pass rate can be calculated in a later step.
+
+// The Build ID needed to fetch the desired data.
+const BUILD_ID = process.env.BUILD_ID;
+// The token need to make the API calls.
+const ADO_API_TOKEN = process.env.ADO_API_TOKEN;
+
+// The workspace where the new files/folder created in this script will be stored.
+const TEST_WORKSPACE = process.env.TEST_WORKSPACE;
```

Comment:
Let's rename this to `BASE_OUTPUT_FOLDER` (both the const in the file and the env variable. `TEST_WORKSPACE` makes sense when we're in ADO, but thinking of this as a standalone script I think this makes clearer what the contents of the variable are. The ADO task will still pass its testWorkspace, but with something like `BASE_OUTPUT_FOLDER: ${{ variables.testWorkspace }}`